extensions/extensions-docker/aissemble-pipeline-invocation/src/main/resources/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi9/openjdk-11-runtime:1.20 AS builder
+FROM registry.access.redhat.com/ubi9/openjdk-17-runtime:1.20 AS builder
```

Comment:
I: Although it's the same image, use a multistage build to save more than 20 mb in space.